frontend/app/src/components/home/Home.svelte

```diff
@@ -334,7 +332,11 @@
                         waitAndScrollToMessageIndex(pathParams.messageIndex, false);
                     }
                 }
-            } else {
+            } else if (
+                pathParams.kind !== "chat_list_route" ||
+                $mobileWidth ||
```

Comment:
This is getting a bit incomprehensible again isn't it?

Reply:
🐁